entity-service/src/test/kotlin/com/egm/stellio/entity/repository/Neo4jRepositoryTests.kt

```diff
@@ -1243,8 +1244,145 @@ class Neo4jRepositoryTests {
         neo4jRepository.deleteEntity(newDevice.id)
     }
 
-    fun createEntity(id: URI, type: List<String>, properties: MutableList<Property> = mutableListOf()): Entity {
-        val entity = Entity(id = id, type = type, properties = properties)
+    @Test
+    fun `it should return an emptyMap for unknown type`() {
+        val entity = createEntity(
+            "urn:ngsi-ld:Beehive:TESTC".toUri(),
+            listOf("https://ontology.eglobalmark.com/apic#Beehive"),
+            mutableListOf(
+                Property(name = "temperature", value = 36),
+                Property(name = "humidity", value = 65)
+            )
+        )
+
+        val attributesInformation = neo4jRepository
+            .getEntityTypeAttributesInformation("https://ontology.eglobalmark.com/apic#unknownType")
+
+        assertTrue(attributesInformation.isEmpty())
+
+        neo4jRepository.deleteEntity(entity.id)
+    }
+
+    @Test
+    fun `it should retrieve entity type attributes information for two entities`() {
+        val firstEntity = createEntity(
+            "urn:ngsi-ld:Sensor:TESTC".toUri(),
```

Comment:
I don't get why all those `Sensor`, `Sensor1`, `Sensor2`,... suddenly appeared.

Reply:
There were conflicts with test data that breaks assertions.
I retried and it seems to work.
I pushed [this](https://github.com/stellio-hub/stellio-context-broker/pull/297/commits/5a0d480365660e6522b70e436666c8dd114cc1d2) fix commit